### PR TITLE
Fix attribute auth

### PR DESCRIPTION
**Pull request by angrykoala**
# Description
Fix auth on attributes

```
Tests improved in branch B:  [
  'Cypher Auth Allow > Read Node & Protected Field',
  'Cypher Auth Allow > Read Relationship & Protected Field',
  'Cypher Auth Roles > Read Node & Cypher Field',
  'Cypher Auth Roles > Read Node & Field',
  'tck/rfs/022 subquery projection > With auth > Nested query'
]
Regressions in branch B:  []
```

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Made some changes to the Neo4j GraphQL Toolbox? Or made some changes to the core library perhaps?

Check out your changes at https://neo4j-graphql-toolbox-3955.surge.sh

Use for instance a local dbms hosted on [Neo4j Desktop](https://neo4j.com/download/) or a cloud-hosted [Aura Free](https://neo4j.com/cloud/aura-free/) dbms.

File: packages/graphql/src/translate/queryAST/factory/OperationFactory.ts [hunk 1]
```diff
@@ -329,6 +351,12 @@ export class OperationsFactory {
         const nodeFields = this.fieldFactory.createFields(target, nodeRawFields, context);
         const edgeFields = this.fieldFactory.createFields(relationship, edgeRawFields, context);
         const authFilters = this.authorizationFactory.createEntityAuthFilters(target, ["READ"], context);
+        // TODO:
```
Comment (darrellwarde) [inline packages/graphql/src/translate/queryAST/factory/OperationFactory.ts:354]: TODO what? TODO WHAT @angrykoala?!?! 😆

**Review by darrellwarde — APPROVED** [review]
LGTM!

File: packages/graphql/src/translate/queryAST/factory/OperationFactory.ts (hunk 1 above)
Comment (angrykoala) [reply packages/graphql/src/translate/queryAST/factory/OperationFactory.ts:354]: :sweat_smile: 

```suggestion
```

**Pull request merged**